app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+
+  def users_active?
```

Comment:
@mauro-oto I don't like the use of the `method_name?` convention here, because it suggests to return a bool value. maybe something more appropriate would be `user_active_class` and just return a `active` string? that would prevent using `raw` as well.